lib/standalone.js

```diff
@@ -41,6 +44,7 @@ async function standalone({
 	customSyntax,
 	cwd = process.cwd(),
 	disableDefaultIgnores,
+	extensions,
```

Comment:
Ill keep this line almost intact: `const extList = extensions?.length ? extensions : DEFAULT_EXTENSIONS;` since `extensions` can be provided but empty. Ill just remove the `?`.

Reply:
Then, how about changing the arguments of `filterByExtensions()` instead of passing a `stylelint` object like this?

```js
if (filePaths.length) {
	filePaths = filterByExtensions(fileList, filePaths, extensions, globCWD);
}

// ...

function filterByExtensions(inputPaths, resolvedFilePaths, extensions, cwd) {
  // ...
}
```